tests/models/test_circulationevent.py

```diff
@@ -106,3 +110,113 @@ def test_new_title(self):
         # The creator of a circulation event is responsible for also
         # updating the dataset.
         eq_(0, event.license_pool.licenses_owned)
+
+    def test_log(self):
+        # Basic test of CirculationEvent.log.
+
+        pool = self._licensepool(edition=None)
+        library = self._default_library
+        event_name = CirculationEvent.DISTRIBUTOR_CHECKOUT
+        old_value = 10
+        new_value = 8
+        start = datetime.datetime(2019, 1, 1)
+        end = datetime.datetime(2019, 1, 2)
+        location = "Westgate Branch"
+
+        m = CirculationEvent.log
+        event, is_new = m(
+            self._db, license_pool=pool, event_name=event_name,
+            library=library, old_value=old_value, new_value=new_value,
+            start=start, end=end, location=location
+        )
+        eq_(True, is_new)
+        eq_(pool, event.license_pool)
+        eq_(library, event.library)
+        eq_(-2, event.delta)  # calculated from old_value and new_value
+        eq_(start, event.start)
+        eq_(end, event.end)
+        eq_(location, location)
```

Comment:
Should these be `eq_(location, event.location)`?

Reply:
Yep.